From 97de2f06ed7f370ffa57de54c57841555be7d6a8 Mon Sep 17 00:00:00 2001
From: Jiafei Pan <Jiafei.Pan@nxp.com>
Date: Mon, 6 Nov 2023 11:35:09 +0800
Subject: [PATCH 24/27] arm64: dts: imx93 rpmsg update generic software mailbox

Adpat to add new property to enable/disable ACK interrupt.

Signed-off-by: Jiafei Pan <Jiafei.Pan@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx93-rpmsg-ca55.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx93-rpmsg-ca55.dtsi b/arch/arm64/boot/dts/freescale/imx93-rpmsg-ca55.dtsi
index a9780af05d71..5db78b628c44 100644
--- a/arch/arm64/boot/dts/freescale/imx93-rpmsg-ca55.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx93-rpmsg-ca55.dtsi
@@ -29,9 +29,9 @@ rpmsg-ca55 {
 		dma-coherent;
 		memory-region = <&vdevbuffer_ca55>;
 		mbox-names = "tx", "rx", "rxdb";
-		mboxes = <&gen_sw_mbox 0 0
-			  &gen_sw_mbox 1 0
-			  &gen_sw_mbox 2 0>;
+		mboxes = <&gen_sw_mbox 0 0 1	/* Tx channel with ACK */
+			  &gen_sw_mbox 1 0 0	/* Rx channel without ACK */
+			  &gen_sw_mbox 2 0 1>;	/* RXDB channel with ACK */
 		vdev-nums = <1>;
 	};
 };
-- 
2.34.1

